data/s3dis/indoor3d_util.py

```diff
@@ -0,0 +1,54 @@
+import glob
+import numpy as np
+import os
```

Comment:
`import os.path as osp`, just as those in other places.

Reply:
Done.